### ISSUE TITLE
dev-java/gradle: do not work with network-sandbox

### ISSUE THREAD

**Issue by tharvik** [gentoo/java-overlay#38]
`dev-java/gradle` do not works with `FEATURES=network-sandbox`.

It might be as easy as modifing `subprojects/docs/src/docs/dsl/org.gradle.api.tasks.wrapper.Wrapper.xml`.

**Comment by chewi** [comment]
@Flowdalic, @msva, please investigate!

**Comment by msva** [comment]
Yes, it is. Just as any other package with "bootsrapping" (where they fetching some crappy dependencies during build time and ignores system-wide installed ones (even if they exists in tree/overlay))

By the way, AFAIK, gradle was work in that way since eternity. Just as maven and some other bootstrap-systems.

Usually, fixing such things requires either heavy patching in ebuild or heavy patching of dist-sources and repackaging.

So, I'll try to take a look, but if you already know a solution, it will be very good if you'll apply a pull-request.

**Comment by chewi** [comment]
Okay, brain wasn't switched on earlier. I am aware that it does this and I don't like it but I was prepared to allow it in java-overlay if fixing it isn't trivial. I wouldn't allow it in the main tree.

**Comment by tharvik** [comment]
#40 should fix it then.

**Comment by Flowdalic** [comment]
What @msva said :)
